templates/rhpam73-authoring-ha.yaml

```diff
@@ -318,17 +318,17 @@ parameters:
   required: true
 ## OpenShift Enhancement BEGIN
 - displayName: Prefer KIE Server OpenShift Service [Tech Preview]
-  description: Enables connection to KIE Server via OpenShift internal Service endpoint (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property)
+  description: If OpenShift Integration of the Business Central is turned on, set this parameter to true to enables connection to KIE Server via an OpenShift internal Service endpoint. Otherwise the Business Central uses a Route to connect to KIE Server. (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property)
   name: KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE
   value: "false"
   required: false
 - displayName: KIE ServerTemplate Cache TTL [Tech Preview]
-  description: KIE ServerTemplate Cache TTL in milliseconds (Sets the org.kie.server.controller.template.cache.ttl system property)
+  description: KIE ServerTemplate Cache TTL in milliseconds. (Sets the org.kie.server.controller.template.cache.ttl system property)
   name: KIE_SERVER_CONTROLLER_TEMPLATE_CACHE_TTL
   value: "60000"
   required: false
-- displayName: Enable OpenShift Integration [Tech Preview]
-  description: If set to true, turns on OpenShift integration feature (Sets the org.kie.workbench.controller.openshift.enabled system property)
+- displayName: Enable OpenShift Integration. [Tech Preview]
+  description: If set to true, turns on OpenShift integration feature in Business Central. With OpenShift integration, Business Central automatically connects to KIE servers in the same namespace, provided that OpenShift integration is enabled on the KIE servers. (Sets the org.kie.workbench.controller.openshift.enabled system property)
```

Comment:
I don't think that "With OpenShift integration, Business Central automatically connects to KIE servers in the same namespace" is a good description, as whether this is enabled or not, the connections will always be in the same namespace (whether they be with the old Controller methodology or the new OpenShiftStartupStrategy). @rhtevan , please suggest a better description?

Reply:
@errantepiphany the operative word here is "automatically". The integration enables *automatic* connection to KIE servers in the same namespace. Should I add "wiithout the need for controller configuration"?

Reply:
@errantepiphany How about this:
_With OpenShift integration enabled, Business Central automatically discovers and connects to KIE Server._

Reply:
@rhtevan sounds perfect to me

Reply:
@mramendi I agree that @rhtevan 's new wording is better.

Reply:
@mramendi Please adding a 'the' --> 'to the KIE Server'